tmt/steps/provision/local.py

```diff
@@ -48,7 +48,7 @@ def requires(self):
         return GuestLocal.requires()
 
 
-class GuestLocal(tmt.Guest):
+class GuestLocal(tmt.GuestSSH):
```

Comment:
I'd prefer to inherit from `tmt.Guest` but define no-op `def stop(self): pass` method

Local uses local execution, not a  ssh session.

Reply:
That makes sense, thanks. `reboot()` also looks like a no-op candidate.

Reply:
And done.